File: yt/yt/flow/examples/java/wait_click_join/wait_click_join/src/main/java/tech/ytsaurus/flow/examples/waitclickjoin/NodeCompanionMain.java
package tech.ytsaurus.flow.examples.waitclickjoin;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.builder.SpringApplicationBuilder;

@SpringBootApplication
public class NodeCompanionMain {
    private static final Logger log = LoggerFactory.getLogger(NodeCompanionMain.class);

    private NodeCompanionMain() {
    }

    // [BEGIN main]
    public static void main(String[] args) throws Exception {
        new SpringApplicationBuilder(NodeCompanionMain.class)
                .run(args);
    }
    // [END main]
}

File: yt/yt/flow/examples/kotlin/wait_click_join/wait_click_join/src/main/kotlin/tech/ytsaurus/flow/examples/waitclickjoin/NodeCompanionMain.kt
package tech.ytsaurus.flow.examples.waitclickjoin

import org.slf4j.Logger
import org.slf4j.LoggerFactory
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.builder.SpringApplicationBuilder

@SpringBootApplication
open class NodeCompanionMain {
    companion object {
        private val log: Logger = LoggerFactory.getLogger(NodeCompanionMain::class.java)

        // [BEGIN main]
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplicationBuilder(NodeCompanionMain::class.java)
                .run(*args)
        }
        // [END main]
    }
}
